docs/source/install/system_requirements.rst

```diff
@@ -22,6 +22,9 @@ be automatically added when you install |st2|.
 +-------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | `Ubuntu 18.04 <http://cdimage.ubuntu.com/releases/18.04.2/release/ubuntu-18.04.2-server-amd64.iso>`_  | `bento/ubuntu-18.04 <https://app.vagrantup.com/bento/boxes/ubuntu-18.04>`_   | `Ubuntu Server 18.04 LTS Bionic  <https://aws.amazon.com/marketplace/pp/B07CQ33QKV/>`_                                                                            |
 +-------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| `RHEL 8 <https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux>`_ /                 | `bento/centos-8.1 <https://app.vagrantup.com/bento/boxes/centos-8.1>`_       | `Red Hat Enterprise Linux (RHEL) 8 (HVM)  <https://aws.amazon.com/marketplace/pp/B07T4SQ5RZ?qid=1581005023484&sr=0-5&ref_=brs_res_product_title>`_                |
```

Comment:
Should probably recommend the official centos vagrant box instead of the bento box version
https://app.vagrantup.com/centos/boxes/8

Reply:
I actually am on the fence about this. I like bento because it supports fusion and virtualbox. And we link to bento for EL7

Reply:
@armab @nmaludy responses to my response?

Reply:
I'm still partial to the CentOS official, but at the end of the day don't really care. I don't think it's a blocker.

Reply:
I'm more in favor of using `centos/8` official Vagrant image too.

However considering `bento` was already mentioned before in this documentation page for EL7 and EL6, - it's OK to me.